### ISSUE TITLE
Site whitelist for fingerprint deception

### ISSUE THREAD

**Issue by ghost** [Eloston/ungoogled-chromium#2714]
### Description

Add an option to disable fingerprint deception for some sites.

### The problem

**Users have no way to disable fingerprint deception for some sites while keeping it enabled for others.**

This issue was originally opened due to the `#fingerprinting-client-rects-noise` flag breaking some sites, such as causing the lines in the Desmos graphing calculator to jitter around randomly at high speeds. That issue has been fixed in #2790, though I had originally thought that the only way to fix it was to disable fingerprint deception entirely.

https://github.com/ungoogled-software/ungoogled-chromium/assets/123329306/f61c3b14-ee8d-43bb-a460-a6ba75ea7215

That issue could previously be reproduced by following these steps in a version prior to the merging of #2790:
- Enable the #fingerprinting-client-rects-noise flag in Ungoogled Chromium (via chrome://flags).
- Go to any Desmos graphing calculator URL, like https://www.desmos.com/calculator/gwqlodfcbm.
- Watch the lines jitter around in a very irritating manner.

### Possible solutions

I suggest adding an option to whitelist some sites from fingerprint deception, which would allow users to disable fingerprinting deception for some sites while keeping it enabled for all other sites. This would allow users to temporarily work around issues that may arise with fingerprint deception breaking sites, as well as issues only pertinent to specific websites.

**Comment by karachikorai** [comment]
@therainydev I am not able to reproduce this error.

**Comment by therainydev** [comment]
@karachikorai
The original issue of the rectangles moving around weirdly was fixed by #2790. (Admittedly, me opening this was a case of the [XY problem](https://xyproblem.info/), thinking the get*ClientRects fingerprint deception itself was the cause.)
I'd like to keep this issue open, though, because it's a separate feature that could be needed (e.g. if sites start detecting fingerprint deception).
Shall I update the original post to make that clear?

**Comment by PF4Public** [comment]
> Shall I update the original post to make that clear?

Yes, please be so kind, more so that one of the issues got fixed. A tight correspondence between issue subject and text would be nice.
